lib/couchbaseX.js

```diff
@@ -73,6 +78,11 @@ class Couchbase extends NoSQL {
     }
     settings.version = settings.version || 5;
 
+    // View.
+    if (settings.stale == couchbase.ViewQuery.Update.BEFORE
+      || settings.stale == couchbase.ViewQuery.Update.AFTER) {
+      this._viewStale = settings.stale;
```

Comment:
can we set blindly as long as the setting.stale is valid?

Reply:
`if ( X or Y) ` is confusing as the reader might start to think why those two are so special.

Reply:
@xavierchow I was trying to prevent the user give an inappropriate value. maybe I could add a value check here.